PSReadLine/Position.cs

```diff
@@ -11,28 +11,17 @@ public partial class PSConsoleReadLine
         /// <param name="current">The position in the current logical line.</param>
         private static int GetBeginningOfLinePos(int current)
         {
-            var newCurrent = current;
-
-            if (_singleton.LineIsMultiLine())
+            int i = Math.Max(0, current);
+            while (i > 0)
             {
-                int i = Math.Max(0, current);
-                while (i > 0)
+                if (_singleton._buffer[--i] == '\n')
                 {
-                    if (_singleton._buffer[--i] == '\n')
-                    {
-                        i += 1;
-                        break;
-                    }
+                    i += 1;
+                    break;
                 }
-
-                newCurrent = i;
-            }
-            else
-            {
-                newCurrent = 0;
             }
 
-            return newCurrent;
+            return i;

```

Comment:
The change to `GetBeginningOfLinePos` is duplicate in #1125, you may remove from this PR.

Reply:
Exactly, just wanted to give you flexibility to accept either PR in the order that you wanted.